SU2_CFD/src/numerics_direct_turbulent.cpp

```diff
@@ -65,39 +65,15 @@ void CUpwScalar::ComputeResidual(su2double *val_residual,
                                        CConfig *config) {
 
   AD::StartPreacc();
-  AD::SetPreaccIn(Normal, nDim);
   AD::SetPreaccIn(TurbVar_i, nVar);  AD::SetPreaccIn(TurbVar_j, nVar);
-  if (grid_movement) {
-    AD::SetPreaccIn(GridVel_i, nDim); AD::SetPreaccIn(GridVel_j, nDim);
-  }
```

Comment:
By not computing the mass flux again we greatly reduce the number of input variables for preaccumulation.